frontend/src/metabase/visualizations/lib/graph/brush.js

```diff
@@ -20,6 +20,11 @@ export function initBrush(parent, child, onBrushChange, onBrushEnd) {
   // the last updated range when brushing
   let range = null;
 
+  // remove deprecated createSVGPoint to fix d3.mouse firefox bug (metabase#24912)
+  if (SVGSVGElement.prototype.createSVGPoint) {
+    SVGSVGElement.prototype.createSVGPoint = undefined;
+  }
```

Comment:
nit: seeing an if block makes me think that this code would behave differently when there's no `createSVGPoint`, but what essentially we want to do here is to remove `createSVGPoint` entirely.

What it ends up behaving is that we want to make `createSVGPoint` `undefined` and if there's no `createSVGPoint` in the first place it will also be `undefined`, so I think it's more concise to drop the if statement since they're equivalent. What do you think?

```suggestion
  SVGSVGElement.prototype.createSVGPoint = undefined;
```

Reply:
It's not equivalent. Changing a `prototype`, even just setting `undefined` to a not-existing property causes some performance issues. That's why it's better to check if we actually need to do this (we don't need for Chrome and Safari).